deployment/live/gcp/conformance/ci/terragrunt.hcl

```diff
@@ -10,6 +10,5 @@ include "root" {
 inputs = merge(
   include.root.locals,
   {
-    server_docker_image = "us-central1-docker.pkg.dev/trillian-tessera/docker-prod/conformance-gcp:latest"
   }
 )
```

Comment:
Is the `merge()` still needed when `server_docker_image` is removed?

Reply:
Not as is, but I left that there in case someone wanted to make overrides specifically for CI.